services/stats/stats.go

```diff
@@ -35,6 +36,9 @@ var enableGCStats bool
 var rc runtimeStatsCollector
 var pkgLogger logger.LoggerI
 var statsSamplingRate float32
+var isMockClient bool
```

Comment:
This is beyond the scope of the PR, but we should avoid using global for those values, they could be part of `HandleT` struct (well `HandleT` should be renamed as well). 

And instead of a global `Setup()`, sync.once could be used, to do lazy init